src/main/java/org/datadog/jenkins/plugins/datadog/listeners/DatadogGraphListener.java

```diff
@@ -103,13 +106,43 @@ public void onNewHead(FlowNode flowNode) {
             TagsUtil.addTagToTags(tags, "stage_depth", String.valueOf(stageDepth));
             // Add custom result tag
             TagsUtil.addTagToTags(tags, "result", result);
+            long pauseDuration = getPauseDuration(startNode);
+
             client.gauge("jenkins.job.stage_duration", getTime(startNode, endNode), hostname, tags);
+            client.gauge("jenkins.job.stage_pause_duration", pauseDuration, hostname, tags);
```

Comment:
Let's make sure to update the metadata.csv for that metric so that it is displayed as ms